lib/browser/objects-registry.js

```diff
@@ -20,6 +20,7 @@ class ObjectsRegistry {
   add(webContents, obj) {
     // Get or assign an ID to the object.
     let id = this.saveToStorage(obj);
+    this.storage[id].count++;
```

Comment:
What about in other cases, where there _are_ overlaps in references?
```
add(webContents1, 1) // set up `browserWindow` for changing size
get(webContents1, 1) // change size to 800x600
add(webContents1, 1) // grab `browserWindow`, `openDevTools()`
remove(webContents1, 1) // dev tools are open, don't need reference any more
get(webContents1, 1) // change size to 128x1024, but reference is now invalid
```

Don't mind the terrible example :wink:
Wouldn't this fail, because `remove()` has been properly called already?